hw3/Advanced_Task_2.ts

```diff
@@ -0,0 +1,25 @@
+//Advanced
+//Task 2
+// Написать программу, которая будет выводить в консоль лесенку.
+// ```
+//     #
+//     ##
+//     ###
+//     ####
+//     #####
+//     ######
+// ```
+// Не забудьте вопспользоваться циклами
+
+let pyramid: string = '';
+const heightOfPyramid: number = 6;            // будет 6 строк
+
+for (let i = 1; i <= heightOfPyramid; i++) { //формируем строки пирамиды
+
+    for (let j = 0; j < i; j++) {            // формируем решетки в строке
+        pyramid += '#';
+    }
+    pyramid += '\n';                         //перенос на новую строку
+}
+
+console.log(pyramid);
```

Comment:
Optimized solution =)
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/47811445/215322295-c1824a8e-8040-498c-a94f-52e1cc9bccb5.png">

Reply:
No way :) Thanks for sharing.